src/Fulfilment/Repository/OrderNotesRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Fulfilment\Repository;
+
+use MyParcelNL\Pdk\Base\Repository\ApiRepository;
+use MyParcelNL\Pdk\Fulfilment\Collection\OrderNoteCollection;
+use MyParcelNL\Pdk\Fulfilment\Request\PostOrderNotesRequest;
+use MyParcelNL\Pdk\Fulfilment\Response\PostOrderNotesResponse;
+
+class OrderNotesRepository extends ApiRepository
+{
+    /**
+     * @param  \MyParcelNL\Pdk\Fulfilment\Collection\OrderNoteCollection $orderNotes
+     * @param  null|string                                               $fulfilmentIdentifier
+     *
+     * @return null|\MyParcelNL\Pdk\Fulfilment\Collection\OrderNoteCollection
+     * @noinspection PhpUnused
+     */
+    public function postOrderNotes(OrderNoteCollection $orderNotes, ?string $fulfilmentIdentifier): ?OrderNoteCollection
+    {
+        // TODO: Check if paid subscription through ACL endpoint
+        if (! $fulfilmentIdentifier) {
+            return null;
```

Comment:
Dit moet altijd null returnen totdat we die subscription functionaliteit erin hebben. dit kan en mag niet gebruikt worden voor die tijd.